src/test/rgw/rgw_multi/tests.py

```diff
@@ -387,6 +395,24 @@ def set_master_zone(zone):
     log.info('Set master zone=%s, waiting %ds for reconfiguration..', zone.name, config.reconfigure_delay)
     time.sleep(config.reconfigure_delay)
 
+def enable_bucket_sync(zone, bucket_name):
+    cmd = ['bucket', 'sync', 'enable', '--bucket', bucket_name] + zone.zone_args()
+    zone.cluster.admin(cmd)
+
+def disable_bucket_sync(zone, bucket_name):
+    cmd = ['bucket', 'sync', 'disable', '--bucket', bucket_name] + zone.zone_args()
+    zone.cluster.admin(cmd)
+
+def check_bucket_sync_status_obj_not_exist(zone, bucket_name):
+    cmd = ['log', 'list'] + zone.zone_arg() + ['|grep ', bucket_name]
```

Comment:
i worry about the `|grep` part here. adding `read_only=True` to the `zone.cluster.admin()` call below makes it fail, because the `--rgw-cache-enabled false` part gets added after the `|`. it does work as is with test_multi.py - i'll have to see if it works in teuthology